pasta.html

```diff
@@ -59,10 +59,9 @@ <h2>Steps to cook Chicken Pasta</h2>
             <li>Serve hot with added garnishing of shredded cheese on the top.</li>
         </ol>
```

Comment:
Please fix the linter error by deleting additional space. You may use [this link](https://github.com/microverseinc/linters-config) to fix all linter errors.